homepage/homepage.html

```diff
@@ -14,8 +14,25 @@ <H1>THE CONGO CHRONICLES</H1>
                 <li><a href="#">Registration</a></li>
                 <li><a href="#">User Profile</a></li>
             </ul>
-        </nav>
+                <hr>
+            </nav>
         </header>
+
+                <nav>
+        <div>
+            <ul>
+                <li><a href="../homepage/homepage.html">HOME</a></li>
+                <li><a href="#">ABOUT</a></li>
+                <li><a href="#">BLOG</a></li>
+                <li><a href="#">CONTACT</a></li> 
+                <li><a href="../submission-page/Submission-page.html">SUBMISSION</a></li>
```

Comment:
I think we should name this "News Submission". "Submission" alone seems vague. What do you think?

Reply:
Well done Everyone.
I agree with Micheal, i think all pages should have uniformity.